docs/automation.md

```diff
@@ -0,0 +1,196 @@
+      resource-mapping:
+        # resource name and the link to original data-source file
+        <resource-name>: <http://source-site.com/datafile.csv>
+
+# the PROCESSING part describes what to do with data, how to 'process' it
+# processors are the tools that will wrangle your data. see:
+# https://github.com/frictionlessdata/datapackage-pipelines - dpp
+# https://github.com/frictionlessdata/tabulator-py - tabulator
+
+# Each processor takes data from "input: <resource-name>",
+# does operations that you define in this section
+# and saves data into "output: <resource-name>",
+# then next processor takes data there and goes on
+# (for now we use the same name for input and output)
+processing:
+  - # put this tabulator processor first in a pipeline if the source is zipped
+    input: <resource-name>
+    tabulator:
+      compression: zip
+    output: <resource-name>
+
+  # Datapackage-pipeline operations example. Here is the dpp docs:
+  # https://github.com/frictionlessdata/datapackage-pipelines
+  -
+    input: <resource-name>
+    dpp:
+      - # delete some columns:
+        run: delete_fields
+        parameters:
+          resources: <resource-name>
+          fields:
+            - id
+            - home_link
+            - keywords
+      - # unpivot table
+        run: unpivot
+        parameters:
+          resources: <resource-name>
+          extraKeyFields:
+            -
+              name: year
+              type: year
+          extraValueField:
+              name: value
+              type: number
+          unpivot:
+            -
+              name: ([0-9]{4})
+              keys:
+                year: \1
+      - # replace, e.g. quarters to dates:  '1998 Q1' -> 1998-03-31 , Q2 -> 06-31, etc
+        run: find_replace
+        parameters:
+          resources: <resource-name>
+          fields:
+            -
+              name: date
+              patterns:
+                -
+                  find: ([0-9]{4})( Q1)
+                  replace: \1-03-31
+                -
+                  find: ([0-9]{4})( Q2)
+                  replace: \1-06-31
+                -
+                  find: ([0-9]{4})( Q3)
+                  replace: \1-09-30
+                -
+                  find: ([0-9]{4})( Q4)
+                  replace: \1-12-31
+    output: <resource-name>
+
+# how often to update the data
+schedule: every 1d
+```
+
+## 3. create `.datahub/datapackage.json`
+
+`.datahub/datapackage.json` file is not required, though it is used to store the dataset metadata (and to describe intermediate resources if you need them) in the package.
+So create `.datahub/datapackage.json` file. Here is example for you:
+
+```json
+{
+  "name": "package_name",
+  "title": "package_title",
+  "description": "description",
+  "license": "ODC-PDDL-1.0",
+  // hope soon the readme will be grabbed automatically
+  "readme": "the very long string of readme, copy of readme.md",
+
+  // other non-necessary fields like:
```

Comment:
maybe "non-required" is a better word here

Reply:
fixed